### ISSUE TITLE
`require("alpha").start` breaking change

### ISSUE THREAD

**Issue by mehalter** [goolord/alpha-nvim#175]
Continuation of https://github.com/goolord/alpha-nvim/issues/174#issuecomment-1421247825

TL;DR With the new changes to the internal structure, `require("alpha").start` requires the configuration to be passed to it rather than knowing what the default configuration is. This breaks the access for users to use this command without manually storing their alpha configuration somewhere in their own user configuration

### Proposed Fix

- store the default configuration from the `setup(...)` call internally in the module
- if `config` is `nil` in `require("alpha").start` then use the internally stored default
- make sure the default config is exported, would be nice for the user to be able to access `require("alpha").config`
